docs-source/content/userguide/managing-domains/model-in-image/usage.md

```diff
@@ -0,0 +1,173 @@
+
+Create additional secrets as needed by macros in your model files. For example, these can store database URLs and credentials that are accessed using `@@SECRET` macros in your model that reference the secrets.  For a discussion of model macros, see [Model files]({{< relref "/userguide/managing-domains/model-in-image/model-files.md" >}}).
+
+#### 6. Domain resource attributes
+
+The following domain resource attributes are specific to Model in Image domains.
+
+| Domain Resource Attribute                    |  Notes |
+| -------------------------                    |  ------------------ |
+| `domainHomeSourceType`                       |  Required. Set to `FromModel`. |
+| `domainHome`                                 |  Must reference an empty or non-existent directory within your image. Do not include the mount path of any persistent volume. Note that Model in Image recreates the domain home for a WebLogic pod every time the pod restarts.|
+| `configuration.model.configMap`             | Optional. Set if you have stored additional models in a config map as per [Optional WDT model config map](#3-optional-wdt-model-config-map). |
+| `configuration.secrets`                      | Optional. Set this array if your image or config map models contain macros that reference custom Kubernetes secrets. For example, if your macros depend on secrets `my-secret` and `my-other-secret`, then set to `[my-secret, my-other-secret]`.|
+| `configuration.model.runtimeEncryptionSecret`| Required. All Model in Image domains must specify a runtime encryption secret. See [Required runtime encryption secret](#4-required-runtime-encryption-secret). |
+| `configuration.model.domainType`             | Set the type of domain. Valid values are `WLS`, `JRF`, and `RestrictedJRF` where `WLS` is the default. See [WDT Domain Types](https://github.com/oracle/weblogic-deploy-tooling/blob/master/site/type_def.md).|
+
+Notes:
+
+ - There are additional attributes that are common to all domain home source types, such as the `image` field. See the Domain Resource [schema](https://github.com/oracle/weblogic-kubernetes-operator/blob/master/docs/domains/Domain.md) and [documentation]({{< relref "/userguide/managing-domains/domain-resource.md" >}}) for a full list of domain resource fields.
+
+ - There are also additional fields that are specific to JRF domain types. For more information, see [Prerequisites for JRF domain types](#7-prerequisites-for-jrf-domain-types).
+
+ - Sample domain resource: For an example of a fully specified sample domain resource, see the the operator source's `kubernetes/samples/scripts/create-weblogic-domain/model-in-image/k8s-domain.yaml.template` file for the [Model in Image]({{< relref "/samples/simple/domains/model-in-image/_index.md" >}}) sample. Note that the `@@` entries in this template are not processed by the operator; they need to replaced with actual values before the resource can be applied.
+
+#### 7. Prerequisites for JRF domain types
+
```

Comment:
It would be helpful to add cross-references between this section and  "> User Guide > Manage FMW Domains > Manage FMW Infrastructure domains" section.

Reply:
I'm hesitant to do this.  The 'Manage FMW Domains' section knows nothing about the MII wallet and secret handling.   In fact, that section should probably be carefully edited to redirect model users to this section.  I'll leave this comment open as a reminder to do so.

Reply:
@doxiao I've filed JIRA OWLS-80732 `Support opss.walletFileSecret and walletPassordSecret in domain-in-image and domain-in-pv` to track this.  Please update the JIRA if you think it needs more info.  Please resolve if you think this is sufficient, or add comments otherwise.

Reply:
Sounds good.